gradle-jdks/src/main/java/com/palantir/gradle/jdks/ToolchainsPlugin.java

```diff
@@ -127,7 +127,7 @@ public void apply(Project rootProject) {
                 .named(LifecycleBasePlugin.CHECK_TASK_NAME)
                 .configure(check -> check.dependsOn(checkJdksLifecycle));
 
-        rootProject.getTasks().register("setupJdks", SetupJdksTask.class, setupJdksTask -> {
+        rootProject.getTasks().register("setupGradleJdks", SetupJdksTask.class, setupJdksTask -> {
```

Comment:
changed the name for consistency

Reply:
do you have to change things in the intellij plugin?

Reply:
Isn't this going to break excavators?

Reply:
> Isn't this going to break excavators?

you are right, thanks for catching this! I reverted this rename. 

> do you have to change things in the intellij plugin?

the intellij plugin is calling directly the `gradle-jdks-setup.sh` script